python/tests/graph/test_utils.py

```diff
@@ -47,23 +47,24 @@ def _gen_invalid_tensor_or_op_with_graph_pairing():
     other_graph = tf.Graph()
     op_name = tnsr.op.name
 
-    # Test get_tensor and get_op returns tensor or op contained in the same graph
+    # Test get_tensor and get_op with non-associated tensor/op and graph inputs
+    _comm_suffix = ' with non-associated tensor/op and graph inputs'
```

Comment:
Renamed the description to clarify the intention of each test case.

Reply:
' with wrong graph' seems good enough here and more readable.

Reply:
just to be clear, my comment was for `_comm_suffix`. the long way to say it for the comment is good since there are now two ways the same thing is described to the user - so it should decrease uncertainty if one of them doesn't quite make sense to the reader.